dev-libs/libgcrypt/libgcrypt-1.11.0.recipe

```diff
@@ -0,0 +1,96 @@
+SUMMARY="GNU's basic cryptographic library"
+DESCRIPTION="Libgcrypt is a general purpose crypto library based on the code \
+used in GnuPG."
+HOMEPAGE="https://gnupg.org/related_software/libgcrypt/"
+COPYRIGHT="2000-2018 Free Software Foundation, Inc."
+LICENSE="GNU LGPL v3"
+REVISION="1"
+SOURCE_URI="https://gnupg.org/ftp/gcrypt/libgcrypt/libgcrypt-$portVersion.tar.bz2"
+CHECKSUM_SHA256="09120c9867ce7f2081d6aaa1775386b98c2f2f246135761aae47d81f58685b9c"
+PATCHES="libgcrypt-$portVersion.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
```

Comment:
Why does this still include a gcc2 patch if gcc2 is disabled?

Also, the secondaryArch package doesn't contain the documentation, this should be cleaned up if there is no primary package on x86 any more.

Reply:
I fixed the gcc2 build of the new version and removed the old one. Having two different versions with the same compatVersion around is problematic because they can't be really distinguished and haikuporter may pick the wrong one otherwise. I learned that the hard way already ;)

Reply:
Thanks! 👍